src/main/java/org/tugraz/sysds/common/Builtins.java

```diff
@@ -106,13 +105,15 @@
 	LSTM_BACKWARD("lstm_backward", false, ReturnType.MULTI_RETURN),
 	LU("lu", false, ReturnType.MULTI_RETURN),
 	MEAN("mean", "avg", false),
+	MICE_LM("mice_lm", true),
```

Comment:
Two modifications are performed in this file
1. declaration of MultiLogReg builtin
2.  ordering of my previous builtins